variables.tf

```diff
@@ -0,0 +1,75 @@
+variable "aggregation_dimensions" {
+  description = "Specifies the dimensions that collected metrics are to be aggregated on."
+  type        = "list"
+
+  default = [
+    ["InstanceId"],
+    ["AutoScalingGroupName"],
+  ]
+}
+
+variable "cpu_resources" {
+  description = "Specifies that per-cpu metrics are to be collected. The only allowed value is *. If you include this field and value, per-cpu metrics are collected"
+  type        = "string"
+  default     = "\"resources\": [\"*\"],"
+}
+
+variable "disk_resources" {
+  description = "If you specify an array of devices, CloudWatch collects metrics from only those devices. Otherwise, metrics for all devices are collected"
+  type        = "list"
+  default     = ["/"]
```

Comment:
is this a folder or URL?

Reply:
This is a mount point. I will fix the description.